targets/targets.json

```diff
@@ -9850,6 +9850,25 @@
         },
         "forced_reset_timeout": 3
     },
+    "S5JS100": {
+        "inherits": ["Target"],
+        "core": "Cortex-M7",
+        "supported_toolchains": ["GCC_ARM", "IAR", "ARMC6"],
+        "default_toolchain": "GCC_ARM",
+        "extra_labels": ["Samsung", "SIDK_S5JS100"],
+        "macros": ["CMSDK_CM7", "MBED_MPU_CUSTOM",  "CMSIS_NVIC_VIRTUAL", "MBEDTLS_CONFIG_HW_SUPPORT"],
+        "device_has": [
+            "INTERRUPTIN",
```

Comment:
Are you planning to add more capabilities like SPI, I2C, etc?

Reply:
Yes. Those will be later included. The Samsung team is targeting the Pelion-Ready Mbed Enablement with SPI, I2C and other features. But for now, this is the first PR of the upcoming and staged PRs. They and I would like to make progressive improvements over time. As you know, having too many features in the first PR is difficult.